Week1/homework/10-compare-arrays.js

```diff
@@ -0,0 +1,25 @@
+//EXERCISE 10 Compare arrays
+
+/*
+Declare 2 variables, that each hold an array. 
+The first array should have 4 items, the second 7 items
+Write a console.log statement that shows the length of each array
+*/
+const WILD_ANIMALS = ["tiger", "elephant", "wolf", "zebra"];
+console.log('The length of the array is '  + WILD_ANIMALS.length + '.');
+const DOMESTIC_ANIMALS = ["dog", "cat", "hen", "horse", "sheep", "cow", "goat"];
```

Comment:
Well done on using meaningful variable names, I see you use all capitals for writing out your `const` variables. 

In most cases this is not necessary, usually I would only assign a number or string to such a const. Reason why is that for instance something should not be able to change.

So `const HOURS_IN_DAY = 24` but an array can despite being a const still receive new values using the `push` method. There are a lot of articles about this subject, for now I would recommend using all capital variable names.